metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/utils/MongoDatabaseUtils.java

```diff
@@ -41,10 +41,14 @@
  *
  * @author Yorgos.Mamakis@ kb.nl
  */
-@SuppressWarnings("rawtypes")
 public class MongoDatabaseUtils<T> {
 
```

Comment:
We could also have a private constructor on this class since everything is static.

Reply:
Fixed.